cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -541,6 +541,108 @@ type ResetConfiguration struct {
 	Timeouts *Timeouts
 }
 
+// UpgradeApplyConfiguration contains a list of configurable options which are specific to the  "kubeadm upgrade apply" command.
+type UpgradeApplyConfiguration struct {
+	// AllowExperimentalUpgrades instructs kubeadm to show unstable versions of Kubernetes as an upgrade
+	// alternative and allows upgrading to an alpha/beta/release candidate version of Kubernetes.
+	AllowExperimentalUpgrades *bool
+
+	// Enable AllowRCUpgrades will show release candidate versions of Kubernetes as an upgrade alternative and
+	// allows upgrading to a release candidate version of Kubernetes.
+	AllowRCUpgrades *bool
+
+	// CertificateRenewal instructs kubeadm to execute certificate renewal during upgrades.
+	CertificateRenewal *bool
+
+	// DryRun tells if the dry run mode is enabled, don't apply any change if it is and just output what would be done.
+	DryRun *bool
+
+	// EtcdUpgrade instructs kubeadm to execute etcd upgrade during upgrades.
+	EtcdUpgrade *bool
+
+	// ForceUpgrade flag instructs kubeadm to upgrade the cluster without prompting for confirmation.
+	ForceUpgrade *bool
+
+	// IgnorePreflightErrors provides a slice of pre-flight errors to be ignored during the upgrade process, e.g. 'IsPrivilegedUser,Swap'.
+	// Value 'all' ignores errors from all checks.
+	IgnorePreflightErrors []string
+
+	// Patches contains options related to applying patches to components deployed by kubeadm during `kubeadm upgrade`.
+	Patches *Patches
+
+	// PrintConfig specifies whether the configuration file that will be used in the upgrade should be printed or not.
+	PrintConfig *bool
```

Comment:
please add the following field to the UpgradeApplyConfiguration

```
	// SkipPhases is a list of phases to skip during command execution.
	// NOTE: This field is currently ignored for "kubeadm upgrade apply", but in the future it will be supported. 
	SkipPhases []string
```

my idea here is to add the field even if "apply" does not support phases yet. in the future it will support them,